python/psim/README.md

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
Might be worth including some brief install instructions, e.g. set up venv and install argparse?

Reply:
This will all come in due time. Probably once we can actually startup a MATLAB simulation with the python configuration parser. Although at least on  my machine `argparse` is already installed. Not sure if this is standard or just happened by chance on my machine.